lib/media/play_rate_controller.js

```diff
@@ -97,7 +110,7 @@ shaka.media.PlayRateController = class {
 
     if (this.rate_ >= 0) {
       try {
-        this.applyRate_(this.rate_);
+        this.applyRate_(this.isBuffering_ ? 0 : this.rate_);
```

Comment:
I'd prefer the form of https://github.com/shaka-project/shaka-player/pull/5696/files#diff-274a4cc3948f85eed18f86e04c6260b93faa97b995d38f0e3b1af618fd01fab7, so I'd prefer add a function to get the rate: `calculateCurrentRate_`

Reply:
I'm pushing an update for it

Reply:
@avelad done